File: modulo1/segmento3/apunte5For/Ejercicio4.java
import java.util.Scanner;

public class Ejercicio4{
	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);

		System.out.print("Ingrese la base deseada: ");
		Integer base= Integer.parseInt(teclado.nextLine());

		System.out.print("Ingrese la altura deseada: ");
		Integer altura= Integer.parseInt(teclado.nextLine());
		
		String linea="";
		
		for(Integer i=0; i<=altura;i++){
			linea="";
			for(Integer j=0;j<=base;j++){
				linea+= "x";
			}
			
			System.out.println(linea);
		}
	
	
	}
}

File: modulo1/segmento3/apunte5For/ejercicio4.py

base = int(input("Ingrese la base deseada: "))
altura = int(input("Ingrese la altura deseada: "))

for i in range(0,altura,1):
	linea=""
	for j in range(0,base,1):
		linea+= "x"

	print(linea)
